typescript/src/lib/modules/lambda_context.ts

```diff
@@ -33,7 +33,19 @@ export class LambdaContext {
                 }
             })
         });
-        const resultExecuteLambda: Result<ExecuteLambdaResponse, Error> = await promiseExecuteLambda;
+        let resultExecuteLambda: Result<ExecuteLambdaResponse, Error>;
+        try {
```

Comment:
pinged you on Slack about why we need to do these. if we really do need these though, we should def extract the `catch` contents out to a helper function, given that we're functionally repeating the same thing everywhere